jubatus/core/classifier/nearest_neighbor_classifier_util.cpp

```diff
@@ -0,0 +1,51 @@
+// Jubatus: Online machine learning framework for distributed environment
```

Comment:
make_id_from_label/get_labe_from_id is  already defined in nearest_neighbor_classifier.cpp.
please remove same function from nearest_neighbor_classifier.cpp and change to call a nearest_neighbor_classifier_util's functions.